import "combat.proto";

package tutorial.orcas.combat.server.event;

// EVENT_COMBAT_BEGIN
message EventCombatBegin {
  required int32 combat_id = 1;                               // 战斗id
}

// EVENT_COMBAT_MOVE_ACTION
message EventCombatMoveAction {
  required int32 combat_id = 1;                               // 战斗id
  required .protocol.CombatAction action = 2;                 // 行动数据
}

// EVENT_COMBAT_BUILD_ACTION
message EventCombatBuildAction {
  required int32 combat_id = 1;                               // 战斗id
  required .protocol.CombatAction action = 2;                 // 行动数据
}
